package.json

```diff
@@ -35,6 +35,7 @@
   ],
   "scripts": {
     "test": "istanbul cover node_modules/mocha/bin/_mocha -- -R spec --require should",
+    "build-lib": "babel src --out-dir lib",
```

Comment:
I guess this should be integrated in the sequential build process kicked-off by "build"; as e.g. `"build:lib": "babel src --out-dir lib",`?